ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/projects.tsx

```diff
@@ -47,7 +43,6 @@ export default function VercelProjects() {
         </div>
         <div className="text-btnPrimary flex cursor-pointer flex-row items-center justify-between text-xs">
           <NewButton
-            onClick={() => router.refresh()}
```

Comment:
How does refresh work after this change?

Reply:
router refetch refetches from the server, refetching data and rerendering.